models/__init__.py

```diff
@@ -1,2 +1,3 @@
 from . import openacademy_course
 from . import openacademy_session
+from . import openacademy_res_partner
```

Comment:
It should be named as model name:

```suggestion
from . import res_partner
```